examples/react/basic-without-generation/package.json

```diff
@@ -10,23 +10,23 @@
     "license-header": "license-header"
   },
   "dependencies": {
+    "@bufbuild/protobuf": "^1.3.1",
+    "@bufbuild/protoc-gen-es": "^1.3.1",
```

Comment:
Shouldn't this be a dev dep? It is fine for example repo probably

Reply:
Yeah it's just an example, and I actually think i'm going to eventually delete this example since it provides nothing useful ontop of the basic version.

Reply:
It's a neat feature that protoc-gen-connect-query isn't strictly necessary. 

`const { say } = createQueryService({ service: ElizaService });` with the ElizaService from protoc-gen-connect-es generated code is functionally equivalent.

But a full example project just for this feature seems a bit much. Just an addition to the docs explaining this alternative would IMO be enough.

Reply:
I have a followup PR that will remove it entirely.